packages/riipen-ui/src/components/ButtonIcon.test.jsx

```diff
@@ -0,0 +1,237 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import ButtonIcon from "./ButtonIcon";
+
+describe("<ButtonIcon>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(<ButtonIcon>children</ButtonIcon>);
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<ButtonIcon>children</ButtonIcon>);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    const wrapper = mount(<ButtonIcon>children</ButtonIcon>);
+
+    const component = wrapper.find("ButtonIcon");
+    expect(component.props().classes).toEqual([]);
+    expect(component.props().color).toEqual("default");
+    expect(component.props().disabled).toEqual(false);
+    expect(component.props().size).toEqual("medium");
+  });
+
+  describe("children prop", () => {
+    it("displays given children", () => {
+      const child = "children";
+
+      const wrapper = mount(<ButtonIcon>{child}</ButtonIcon>);
+
+      expect(wrapper.find("ButtonIcon").contains(child)).toEqual(true);
+    });
+  });
+
+  describe("classes prop", () => {
+    it("applies custom classes", () => {
+      const classes = ["classOne"];
+
+      const wrapper = mount(
+        <ButtonIcon classes={classes}>children</ButtonIcon>
+      );
+
+      expect(
+        wrapper
+          .find("ButtonIcon")
+          .childAt(0)
+          .hasClass(classes[0])
+      ).toEqual(true);
+    });
+  });
+
+  describe("color prop", () => {
+    it("sets color class name with a valid color", () => {
+      const color = "muted";
+
+      const wrapper = mount(<ButtonIcon color={color}>children</ButtonIcon>);
+
+      expect(
+        wrapper
+          .find("ButtonIcon")
+          .childAt(0)
+          .hasClass(color)
+      ).toEqual(true);
+    });
+
+    it("gives an error when given an invalid color", () => {
+      const errors = jest.spyOn(console, "error").mockImplementation();
+
+      mount(<ButtonIcon color="pink">children</ButtonIcon>);
+
+      expect(errors).toHaveBeenCalledTimes(1);
+    });
+  });
+
+  describe("component prop", () => {
+    it("sets component with valid custom component", () => {
+      const component = "span";
+
+      const wrapper = mount(
+        <ButtonIcon component={component}>children</ButtonIcon>
+      );
+
+      expect(
+        wrapper
+          .find("ButtonIcon")
+          .childAt(0)
+          .name()
+      ).toEqual(component);
+    });
+
+    it("sets component with 'button' when component prop is not provided", () => {
```

Comment:
Could this be rephrased to something like...

"sets component type to button when component prop is not provided"